src/main/java/com/psk/pyragas/ToolRent/usecases/Advertisements.java

```diff
@@ -8,19 +8,27 @@
 import lombok.Setter;
 
 import javax.annotation.PostConstruct;
-import javax.enterprise.inject.Model;
+import javax.faces.context.FacesContext;
+import javax.faces.view.ViewScoped;
 import javax.inject.Inject;
+import javax.inject.Named;
 import javax.transaction.Transactional;
+import java.io.Serializable;
 import java.util.List;
+import java.util.Map;
 
-@Model
 @Getter
 @Setter
-public class Advertisements {
+@Named
+@ViewScoped
+public class Advertisements implements Serializable {
```

Comment:
Maybe we should name all use cases that are ViewScoped with a "View" at the end of the class name? Or make a package? Scopes are easy to miss, forget. In xhtml we only type the name, so it would make sense to write something like this in the name of the class.